module/Activity/src/Activity/Service/Activity.php

```diff
@@ -32,6 +32,17 @@ protected function getDefaultResourceId()
     }
 
     /**
+     * Return the form for this activity
+     *
+     * @return ActivityForm
+     */
+    public function getForm()
+    {
+        $this->permissionCreateActivityOrException();
+        return new ActivityForm();
```

Comment:
It is better to obtain the form from the Service Manager. Such that you can inject things like translation from there.